build.wake

```diff
@@ -115,83 +142,112 @@ def setVersion release file =
 def tarball Unit =
     def releaseResult = buildAs Unit
     def timeResult = buildOn Unit
+
     require Pass release = releaseResult
     require Pass time = timeResult
+
     # Create debian + RedHat package files
-    require Pass changelog =
-        setVersion release "debian/changelog"
-    require Pass spec =
-        setVersion release "wake.spec"
+    require Pass changelog = setVersion release "debian/changelog"
+    require Pass spec = setVersion release "wake.spec"
+
     # Identify those sources files to include in the tarball
-    require Pass allSources =
-        sources "." `.*`
+    require Pass allSources = sources "." `.*`
+
     def srcs =
         allSources
-        | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
-        | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
-        | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
+        | filter (!matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
+        | filter (!matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
+        | filter (!matches `(.*/)*\.gitignore` _.getPathName) # omit git files
```

Comment:
I think it might actively be a problem. If we do add an exception it should be for when the child is a primitive expression (parens, numbers, strings, variables, etc...)

Reply:
https://github.com/sifive/wake/pull/1232

Reply:
That only applied to `!`, please lmk if there are other unary op that is should apply to (with code examples please)